src/Longitude.cpp

```diff
@@ -102,15 +107,17 @@ const char *Longitude::formatStringForMeade(char *targetBuffer) const
     secs      = secs - mins * 60;
     LOG(DEBUG_MEADE, "[LONGITUDE] Degs is %l, Mins is %l", degs, mins);
 
-    // Since internal storage is actual longitude, Meade is negated
+    char sgn;
     if (totalSeconds > 0)
     {
-        // Since we already inverted it when it was negative (by using ABS a few
-        // lines above here), we only invert it if it is positive.
-        degs = -degs;
+        sgn = '-';
```

Comment:
I think I got something wrong here.. when I checked the INDI logs I spotted 
Longitude 00:33:0.0 (0.55) (Longitude sign in carthography format) but this is not correct for my location .. the carthography longitude is -00:33:0 .. I am west of the prime meridian.  OAT does get the correct longitude it shows on the LCD display 0.54W 

I will do a bit more testing!

Reply:
well I did a quick test with all the debug on for INDI and this is what I get 
SCOPE	88.941888 sec	: <setSiteLongitude>
SCOPE	88.941930 sec	: CMD <:Sg000*32#>
SCOPE	88.964877 sec	: CMD <:Sg000*32#> successful.
....
INFO	88.986112 sec	: Site location in the mount updated to Latitude xxxxx Longitude  0:32:00.0 (0.533333) (Longitude sign in carthography format)
DEBUG	88.987411 sec	: Configuration successfully saved for GEOGRAPHIC_COORD.
INFO	88.987441 sec	: Observer location updated: Latitude xxxx Longitude  0:32:00.0 (0.53)

So my conclusion for this is that the INDI log statement might be incorrect since it is sending over the wire the correct west positive longitude.

I checked in skychart and it is correctly showing the longitude as -0:32:00 ie West 32 degrees and that is correct.